cmd/node/node.go

```diff
@@ -399,14 +399,17 @@ func (app *App) setupGenesis(state *state.TransactionProcessor, msh *mesh.Mesh)
 	for id, balance := range app.Config.Genesis.Accounts {
 		bytes := util.FromHex(id)
 		if len(bytes) == 0 {
-			return fmt.Errorf("cannot read config entry for genesis account %s", id)
+			return fmt.Errorf("cannot decode entry %s for genesis account", id)
+		}
+		// just make it explicit that we want address and not a public key
+		if len(bytes) > types.AddressLength {
```

Comment:
`!=` to be precise?

Reply:
there was 0x1 in default config file, and i didn't want to add 19 zeroes but i think i can remove it completely